Miniscript-Template-000-Timelocks-in-Templates.mediawiki

```diff
@@ -0,0 +1,83 @@
+<pre>
+  Miniscript Template: 0
```

Comment:
I recommend the front matter be

MSTemplate: 0

similar to the no space file naming...

Reply:
Agreed, I actually think we can compress it further MST for shorter filenames.